rotkehlchen/api/v1/schemas.py

```diff
@@ -197,11 +216,18 @@ def validate_ethtx_query_schema(  # pylint: disable=no-self-use
             data: Dict[str, Any],
             **_kwargs: Any,
     ) -> None:
-        value = data['order_by_attribute']
-        if data['order_by_attribute'] not in (None, 'timestamp'):
+        valid_ordering_attr = {None, 'timestamp'}
+        if (
+            data['order_by_attributes'] is not None and
+            not set(data['order_by_attributes']).issubset(valid_ordering_attr)
+        ):
+            error_msg = (
+                f'order_by_attributes for transactions can not be '
+                f'{",".join(set(data["order_by_attributes"]) - valid_ordering_attr)}'
```

Comment:
nitpick: I prefer the function names to overloaded operators. It's much more explicit and easier to understand here and below
```suggestion
                f'{",".join(set(data["order_by_attributes"]).difference(valid_ordering_attr))}'
```